packages/gatsby-transformer-yaml/package.json

```
@@ -8,7 +8,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.0.0",
-    "js-yaml": "3.11.0",
+    "js-yaml": "3.13.0",
```

I would honestly move to a caret as it's been fixed from the start https://github.com/alampros/gatsby/commit/3b6f6555bfe74b20ab3e99f2f2fb1a4c8be78aa7 which I'm unsure why as well but lets just hope that semver won't break on js-yaml and go with the regular approach.
```suggestion
    "js-yaml": "^3.13.0",
```

---

For sure! Thank you!